file_picker/plugin.go

```diff
@@ -0,0 +1,62 @@
+package file_picker
+
+import (
+	"github.com/go-flutter-desktop/go-flutter"
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+	"github.com/pkg/errors"
+)
+
+const channelName = "plugins.flutter.io/file_picker"
+
+type FilePickerPlugin struct {
+	VendorName      string
+	ApplicationName string
```

Comment:
Again here, VendorName and ApplicationName are not used by the FilePickerPlugin and should be removed.